### ISSUE TITLE
Docs for running an a11y test

### ISSUE THREAD

**Issue by Garbee** [puppeteer/puppeteer#104]
Found an [example running Axe](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Quick-Start:-Running-an-Audit-with-Web-Puppeteer) for a11y tests using the old Google Puppeteer library. This looks like prime meat to get ported with an example of running it with this library instead.

I'll work on this when I get the time later this week to write a few other examples for the project.

**Comment by aslushnikov** [comment]
What's the status of this? Shall we rather have the issue to be filed in https://github.com/GoogleChrome/accessibility-developer-tools?

**Comment by Garbee** [comment]
I have a basic outline in a local file. I'll find it and put it into a Google Doc as I work on filling in the content later today.

My main goal is to show an example of using Axe directly to do a test and get the results. The a11y-devtools repo seems to me like it is a wrapper in itself to various browser drivers already. If that is correct, then another task should be made there post-launch (or pre-launch privately) for an update to use Puppeteer to drive Chrome Headless.

**Comment by Garbee** [comment]
What I'm going to do is get a doc ready (hopefully within a day or two have it done) and once this project soft launches I'll post it to my personal blog to start. If we want to get the content anywhere else I'm open to porting it for anyone.

So, closing this up and we can handle content placement as an auxiliary thing later.
